Bullwinkle.class.nut

```diff
@@ -442,6 +448,14 @@ class Bullwinkle {
                 }
             }
         }
+
+        // If packages still pending schedule next run
+        if ( _packages.len() > 0 ) {
+            _watchdogTimer = imp.wakeup(0.5, _watchdog.bindenv(this));
```

Comment:
It certainly wouldn't hurt to have a constant.

I'm not even sure why 0.5 was chosen. The timeout is in seconds so I would have thought a value of 1 would be more logical.